docs/use_cases/tooltip.html

```diff
@@ -0,0 +1,240 @@
+          <p class="card-text">
+            This function is activated by configuring the graph options as follows:
+            <code>
+              <pre>
+tooltip: {
+    confine: true,
+}
+              </pre>
+            </code>
+          </p>
+          <div id="htmlId">
+            <div class="border border-light position-relative">
+              <div class="chart_title">
+                <h4 class="display-4 mx-3 mb-1 mt-3">Title</h4>
+                <h5 class="display-5 mx-3 mb-1 mt-0">Sub-Title</h5>
+              </div>
+              <div id="barLine_holder">
+                <div id="barLine_chart" style="width: 100%; height: 50vh" class="position-relative"></div>
+              </div>
+              <div id="barLine_legend"></div>
+            </div>
+          </div>
+          <script>
+            addViewCode();
+          </script>
+        </div>
+      </div>
+      <script id="codeId">
+        ///////////////////////////////////////////////////
+        // Used data
+        ///////////////////////////////////////////////////
+
+        // Data to be displayed
+        var dataOptions = {
+          grid: {
+            left: '0%',
+            right: '0%',
+          },
+          xAxis: {
+            type: 'category',
+            data: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun'],
+          },
+          yAxis: {},
+          series: [
+            {
+              data: [10, 22, 28.8956454657, 23, 19, 15],
+              type: 'bar',
+            },
+            {
+              data: [12, 28.8956454657, 23, 15, 15, 18],
+              type: 'line',
+            },
+          ],
+          legend: {
+            data: ['label 0', 'label 1'],
+          },
+          tooltip: {
+            confine: true,
+          },
+        };
+
+        ///////////////////////////////////////////////////
+        // ODS Charts
+        ///////////////////////////////////////////////////
+        // Build the theme
+        var themeManager = ODSCharts.getThemeManager();
+        echarts.registerTheme(themeManager.name, themeManager.theme);
+
+        // Get the chart holder and initiate it with the generated theme
+        var div = document.getElementById('barLine_chart');
+        var myChart = echarts.init(div, themeManager.name, {
+          renderer: 'svg',
+        });
+
+        // Set the data to be displayed.
+        themeManager.setDataOptions(dataOptions);
+        // Register the externalization of the legend.
+        themeManager.externalizeLegends(myChart, '#barLine_legend');
+        // Manage window size changed
+        themeManager.manageChartResize(myChart, 'barLine_chart');
+        // Register the externalization of the tooltip/popup
+        themeManager.externalizePopover();
+        // Display the chart using the configured theme and data.
+        myChart.setOption(themeManager.getChartOptions());
+      </script>
+    </div>
+
+    <div class="container pt-3">
```

Comment:
Just a reminder for later as well, but it might be great to use `.container-xxl` here instead of `.container`